test/support/models.rb

```diff
@@ -128,6 +128,7 @@ def column_for_attribute(attribute)
 
   begin
     require 'active_model/type'
+    require 'active_record/type'
```

Comment:
I was looking at this run on travis
https://travis-ci.org/plataformatec/simple_form/jobs/226925127

that failed for that reason.